https://sourceforge.net/p/itpp/git/merge-requests/3/

Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
Date:   Wed Sep 13 18:33:01 2017 +0200 

    use cmake TIMESTAMP
    
    to make build more platform-independent
    and to allow to override build date to enable reproducible builds
     
    See https://reproducible-builds.org/ for why this is good.
    
    Also use UTC to be independent of timezones.

Index: itpp-4.3.1/CMakeLists.txt
===================================================================
--- itpp-4.3.1.orig/CMakeLists.txt
+++ itpp-4.3.1/CMakeLists.txt
@@ -25,7 +25,7 @@
 #
 # -------------------------------------------------------------------------
 
-cmake_minimum_required ( VERSION 2.8.6 )
+cmake_minimum_required ( VERSION 2.8.11 )
 
 set ( CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules/ ${CMAKE_MODULE_PATH} )
 
@@ -154,12 +154,7 @@ set (PACKAGE_VERSION "4.3.1")
 set (PACKAGE_STRING "${PACKAGE_NAME} ${PACKAGE_VERSION}")
 set (PACKAGE_TARNAME ${PACKAGE})
 set (PACKAGE_URL "")
-if (UNIX)
-  EXECUTE_PROCESS(COMMAND "date" "+%d/%m/%Y" OUTPUT_VARIABLE PACKAGE_DATE)
-  string(REGEX REPLACE "(..)/(..)/..(..).*" "\\1/\\2/\\3" PACKAGE_DATE "${PACKAGE_DATE}")
-else()
-  set(PACKAGE_DATE "")
-endif()
+string(TIMESTAMP PACKAGE_DATE "%Y-%m-%d" UTC)
 
 if (WIN32 AND NOT MINGW)
   #TODO: remove this from future versions
